### ISSUE TITLE
keep feed reading position when toggling

### ISSUE THREAD

**Issue by VivienJustJ** [elliottetzkorn/t4t#136]
When scrolling posts, if you toggle to messages or reactions, then back to posts, it returns you to the most recent post.

It might be better if it returned you to your previous position, and only brought you to the top when tapping the feed icon IN the feed.

**Comment by VivienJustJ** [comment]
Testing this, I find it works fine when toggling to messages, even into and out of individual messages, then returning to the feed. 

Toggling to reacts and profile return to the top of the feed.

**Comment by elliottetzkorn** [comment]
Yeah, this is true, and has to do with how Flutter handles memory. There is a hack to force it to keep position in memory, which I could implement!

**Comment by VivienJustJ** [comment]
please do! it was not an issue when i was seeing only 6 posts per day, but now there's at least 12 -15!
